includes/api/class-wc-rest-orders-controller.php

```diff
@@ -257,7 +257,7 @@ public function get_collection_params() {
 
 		$params['status'] = array(
 			'default'           => 'any',
-			'description'       => __( 'Limit result set to orders assigned a specific status.', 'woocommerce' ),
+			'description'       => __( 'Limit result set to orders assigned to specific statuses.', 'woocommerce' ),
```

Comment:
I think the wording is a bit unusual, orders are not assigned to statuses. I would use sth like 'Limit result set to orders assigned one of the specified statuses.' or 'Limit result set to orders having specific statuses assigned'?
@mikejolley can you please help as a native speaker?

Reply:
..well it's not bad just a bit of a mouthful. I'd go with:

```
List orders which have specific statuses.
```

Reply:
@mikejolley `Limit result set to` is like a standard on the REST API: https://developer.wordpress.org/rest-api/reference/posts/#arguments

Reply:
Let's go with `Limit result set to orders which have specific statuses` then, right?